Add spellcheck & remediate found spelling issues

SeanKilleen opened the pull request:
<!--
Thank you for contributing to this project! You must fill out the information below before we can review this pull request. By explaining why you're making a change (or linking to an issue) and what changes you've made, we can triage your pull request to the best possible team for review.
-->

### Why:

Closes #23188

<!-- If there's an existing issue for your change, please replace ISSUE above with a link to the issue.
If there's _not_ an existing issue, please open one first to make it more likely that this update will be accepted: https://github.com/github/docs/issues/new/choose. -->

### What's being changed (if available, include any code snippets, screenshots, or gifs):

* Add GitHub Action that runs cSpell (which the devcontainer already uses as an extension)
* Remediate cSpell findings by fixing issues, adding ignores, or adding words to the dictionary.
* Fix ~20 spelling issues.

<!-- Let us know what you are changing. Share anything that could provide the most context.
If you made changes to the `content` directory, a table will populate in a comment below with links to the preview and current production articles. -->

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).


## Notes on Remediations

* In some cases, I noticed issues raised because liquid syntax was missing a space at the end. In those cases, I fixed the liquid syntax
* In cases where the cSpell finding was related to terminology within the body of a document, I added back-ticks in cases where I thought that choice would be aligned with the markdown standards.
* In cases where words were used in one article only, I added `cSpell:ignore` lines to the frontmatter
* Oftentimes I added words to the cSpell dictionary to remediate.
* cSpell uses en-US by default; in some cases I changed words to conform to that standard (e.g. `behaviour` to `behavior`).

## Remaining Tasks

Surfaced by the build system that I can see.

* [x] Reconcile cSpell's frontmatter ignore syntax with the link-check that is failing due to invalid frontmatter
* [x] Run prettier on the `spellcheck.yml` file
* [x] Understand why unit tests are failing on the contents of the spellcheck workflow and remediate
* [ ] Remove workflow entirely if it's not allowed (hopefully not the case)
* [ ] Squash changes (if maintainers won't be doing that)

SeanKilleen committed:
add cspell config

SeanKilleen committed:
add github action

SeanKilleen committed:
Add optional dash to liquid syntax

Some of the docs have this, though I'm not sure why.

SeanKilleen committed:
fix liquid dash syntax

SeanKilleen committed:
fix: backticks

SeanKilleen committed:
ignore in file

SeanKilleen committed:
ignores

SeanKilleen committed:
ignore in file

SeanKilleen committed:
ignore in file

SeanKilleen committed:
dictionary adds

SeanKilleen committed:
more adds

SeanKilleen committed:
bacticks

SeanKilleen committed:
typo

SeanKilleen committed:
add words

SeanKilleen committed:
ignore in files

SeanKilleen committed:
ignore in a file

SeanKilleen committed:
add words

SeanKilleen committed:
Add words

SeanKilleen committed:
fix: add dash

SeanKilleen committed:
fix: add dash

SeanKilleen committed:
ignore in file

SeanKilleen committed:
add more words

SeanKilleen committed:
fix: add code fence

SeanKilleen committed:
fix typo

SeanKilleen committed:
fix: dash

SeanKilleen committed:
add words

SeanKilleen committed:
Add words

SeanKilleen committed:
fix: uses dashes with "un"

SeanKilleen committed:
Fix: backticks

SeanKilleen committed:
Add words

SeanKilleen committed:
fix: close code block

SeanKilleen committed:
ignore in file

SeanKilleen committed:
Revert "fix: uses dashes with "un""

This reverts commit 0694b03fde4b1ec1a5074cd79dd2825aebf010ff.

SeanKilleen committed:
Add words

SeanKilleen committed:
fix my fix :)

SeanKilleen committed:
inline ignore

SeanKilleen committed:
in-line ignores

SeanKilleen committed:
Add words

SeanKilleen committed:
disable cSpell in libraries file

SeanKilleen committed:
Add words

SeanKilleen committed:
inline ignores

SeanKilleen committed:
Add words

SeanKilleen committed:
Add words

SeanKilleen committed:
Add words

SeanKilleen committed:
Add words

SeanKilleen committed:
in-line ignores

SeanKilleen committed:
Add words

SeanKilleen committed:
Add a link

SeanKilleen committed:
inline ignore

SeanKilleen committed:
Add word

SeanKilleen committed:
fix: Add space in liquid syntax

The non-standard liquid was defeating the regex ignore.

SeanKilleen committed:
fix: typo

SeanKilleen committed:
Add words

SeanKilleen committed:
fix: add space

SeanKilleen committed:
add word

SeanKilleen committed:
Additional ignore for liquid with dash at end

SeanKilleen committed:
fix: liquid spacing

SeanKilleen committed:
Add

SeanKilleen committed:
fix typo

SeanKilleen committed:
fix typo

SeanKilleen committed:
ignores

SeanKilleen committed:
Add words

SeanKilleen committed:
fix

SeanKilleen committed:
Add words

SeanKilleen committed:
fix

SeanKilleen committed:
ignore instead of fix

SeanKilleen committed:
backticks

SeanKilleen committed:
fixes

SeanKilleen committed:
Add word

SeanKilleen committed:
ignore in file

SeanKilleen committed:
Add words

SeanKilleen committed:
local ignores

SeanKilleen committed:
Add words

SeanKilleen committed:
fix spacing

SeanKilleen committed:
fix

SeanKilleen committed:
fix

SeanKilleen committed:
add words

SeanKilleen committed:
Capitalization of JetBrains

SeanKilleen committed:
Add word

SeanKilleen committed:
local ignore

SeanKilleen committed:
standardize on en-US

SeanKilleen committed:
Words

SeanKilleen committed:
local ignores

SeanKilleen committed:
fix

SeanKilleen committed:
Add words

SeanKilleen committed:
fix

SeanKilleen committed:
more

SeanKilleen committed:
fix

SeanKilleen committed:
Add words

SeanKilleen committed:
local ignores

SeanKilleen committed:
Add

SeanKilleen committed:
local ignores

SeanKilleen committed:
Ignore for both dashes

SeanKilleen committed:
fi

SeanKilleen committed:
backticks

SeanKilleen committed:
local ignore

SeanKilleen committed:
spacing

SeanKilleen committed:
add words

SeanKilleen committed:
local ignore

SeanKilleen committed:
Add a word

SeanKilleen committed:
add words

SeanKilleen committed:
en-us

SeanKilleen committed:
spacing

SeanKilleen committed:
add

SeanKilleen committed:
spacing

SeanKilleen committed:
Adds

SeanKilleen committed:
more

SeanKilleen committed:
more

SeanKilleen committed:
local ignore

SeanKilleen committed:
more

SeanKilleen committed:
fix

SeanKilleen committed:
Add wordss

SeanKilleen committed:
more

SeanKilleen committed:
more

SeanKilleen committed:
more

SeanKilleen committed:
more

SeanKilleen committed:
more

SeanKilleen committed:
more

SeanKilleen committed:
consolidate dashes for liquid with regex

SeanKilleen committed:
fix the actions file

SeanKilleen committed:
Merge branch 'main' into add-spellcheck

SeanKilleen commented on content/site-policy/site-policy-deprecated/github-ae-data-protection-agreement.md:
```diff
@@ -118,7 +119,7 @@ No Use in Marketing and CCPA. GitHub will not use the Protected Data for the pur
 
 #### 6.2 Acceptance of GitHub Subprocessors.
-Customer authorizes GitHub and its subprocesors to appoint and use subprocessors in accordance with this Section 6 and the Agreement.
+Customer authorizes GitHub and its subprocessors to appoint and use subprocessors in accordance with this Section 6 and the Agreement.
```
⚠️ Calling attention to this change in particular. As far as I can tell, it's a definite typo, but I want to ensure there are no legal implications for changing the typo here in the text of a legal agreement.

SeanKilleen commented:
😕 I'm a little confused as to why a new GitHub Action that introduces new functionality (and is running successfully in this PR) can't be considered for a contribution. I'd like a human to explain this before I remove it, since the GitHub action is what will provide the ongoing benefit.

SeanKilleen committed:
update the format of the YAML file

SeanKilleen committed:
remove cSpell:ignore statements

SeanKilleen committed:
move ignore words into cSpell.json

SeanKilleen committed:
remove another cspell frontmatter entry

SeanKilleen committed:
Merge branch 'main' into add-spellcheck

SeanKilleen committed:
ignore the words in cSpell.json

SeanKilleen commented:
Summary of where things stand: 

✅ I have spot-checked at least one link for every page
✅ Builds are mostly succeeding
✅ GitHub Action is succeeding
⚠️ Unit test failure seems to indicate that I may need to change up my GitHub Action to use a different runner/container. Which is fine, I can investigate.
⚠️ Need clarity on why it's not possible to accept a workflow YAML file as a contribution if it serves the purpose. But if someone can explain that to me as necessary, I can drop off the GitHub action

SeanKilleen committed:
Use common commit of actions

SeanKilleen committed:
Merge branch 'main' into add-spellcheck

SeanKilleen commented on content/copilot/configuring-github-copilot/configuring-github-copilot-in-a-jetbrains-ide.md:
```diff
@@ -78,13 +78,13 @@ You can manage advanced settings for {% data variables.product.prodname_copilot
 1. Under **Languages & Frameworks**, click **{% data variables.product.prodname_copilot %}**.
 1. Edit the settings according to your personal preferences.
-   - To adjust the behaviour and appearance of code suggestions, and whether to automatically check for updates, select or deselect the corresponding checkboxes.
+   - To adjust the behavior and appearance of code suggestions, and whether to automatically check for updates, select or deselect the corresponding checkboxes.
```
ℹ️ Note: cSpell defaults to en-US, and I think the docs do too, so I changed the few instances of this that I found.

SeanKilleen commented on content/repositories/working-with-files/using-files/viewing-a-file.md:
```diff
@@ -47,7 +47,7 @@ In a file or pull request, you can also use the {% octicon "kebab-horizontal" ar
 3. In the upper-right corner of the file view, click **Blame** to open the blame view.
 ![Blame button](/assets/images/help/repository/blame-button.png)
-4. To see earlier revisions of a specific line, or reblame, click {% octicon "versions" aria-label="The prior blame icon" %} until you've found the changes you're interested in viewing.
+4. To see earlier revisions of a specific line, or re-blame, click {% octicon "versions" aria-label="The prior blame icon" %} until you've found the changes you're interested in viewing.
```
ℹ️ This may be preference; if you prefer the non-hyphenated version, I'll revert this change and add `reblame` to the cSpell dictionary.

SeanKilleen commented on content/site-policy/acceptable-use-policies/github-bullying-and-harassment.md:
```diff
@@ -20,4 +20,4 @@ We do not tolerate harassment, bullying, or abuse of any kind, whether directly
 - Creating alternative accounts specifically to evade moderation action taken by GitHub staff or users
 
-Please note, not all unwelcome conduct is necessarily considered harassment. For example, disagreeing with another user or downvoting their comments may not rise to the level of harassment on our platform. In addition, sharing criticism of public figures or projects, or topics of public interest, does not necessarily fall under this policy. However, we encourage you to be mindful in how you engag ...
+Please note, not all unwelcome conduct is necessarily considered harassment. For example, disagreeing with another user or down-voting their comments may not rise to the level of harassment on our platform. In addition, sharing criticism of public figures or projects, or topics of public interest, does not necessarily fall under this policy. However, we encourage you to be mindful in how you enga ...
```
ℹ️ Same here; if there's a preference for the non-hyphenated version this should be a quick revert.

SeanKilleen committed:
Merge branch 'main' into add-spellcheck

cmwilson21 commented:
@SeanKilleen Thanks for opening a PR and linking it to your issue! ✨ 

We're going to take a look at this internally and I'll get back to you 💖

SeanKilleen commented:
@cmwilson21 sounds good! The GitHub action is super basic and I've pinned the actions to the same commit hashes used by the other actions in the project, so I'm hoping it can still be useful. Worst case, I can remove it and we can merge in all the spelling fixes, but the real value is in having it there long-term.

I'm also happy to put together some text in a README on how it works / how to take certain actions like adding words to the dictionary. Just let me know where you want it. 👍

Thanks for taking a look!

martinwoodward commented:
WOW 😍 

First of all, LOVE this approach.  You've got a really long way with that cspell integration and looks amazingly close. We're going to need to have a bit of a think how we integrate this approach with the backend repos that this links into so that might take a while to bring in and might not exactly be this way but it's an **really** compelling approach.  Thank you so much for all that work!!

While we figure that out, it would be great to create a new PR that just brings in some of the low-hanging fruit content changes that this identified, specifically these files:

- [ ] content/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/about-your-organizations-profile.md
- [ ] content/admin/monitoring-activity-in-your-enterprise/reviewing-audit-logs-for-your-enterprise/audit-log-events-for-your-enterprise.md (Line 611)
- [ ] content/billing/managing-billing-for-github-advanced-security/about-billing-for-github-advanced-security.md
- [ ] content/code-security/dependabot/dependabot-version-updates/configuring-dependabot-version-updates.md
- [ ] content/code-security/security-advisories/guidance-on-reporting-and-writing/privately-reporting-a-security-vulnerability.md
- [ ] content/code-security/security-advisories/repository-security-advisories/configuring-private-vulnerability-reporting-for-a-repository.md
- [ ] content/codespaces/customizing-your-codespace/configuring-automatic-deletion-of-your-codespaces.md
- [ ] content/codespaces/developing-in-codespaces/stopping-and-starting-a-codespace.md
- [ ] content/codespaces/managing-codespaces-for-your-organization/restricting-access-to-machine-types.md
- [ ] content/copilot/configuring-github-copilot/configuring-github-copilot-in-a-jetbrains-ide.md
- [ ] content/copilot/troubleshooting-github-copilot/troubleshooting-github-copilot-in-a-jetbrains-ide.md (lines 14 and 18)
- [ ] content/get-started/onboarding/getting-started-with-github-enterprise-cloud.md
- [ ] content/get-started/using-github/github-command-palette.md
- [ ] content/packages/learn-github-packages/deleting-and-restoring-a-package.md
- [ ] content/rest/commits/comments.md (line 17)

Are you happy to create that separate PR @SeanKilleen or do you want me to do it (giving credit to you obviously)?

SeanKilleen commented:
@martinwoodward sure thing! I understand that I don't have full visibility. 👍

Happy to create a separate PR for the content changes and have this be just for the GitHub action. 👍 I'm a little tied up today but I'll get to it tonight or over the weekend.

Thanks for having a look! Sorry to have pinged you about it a little prematurely.

martinwoodward commented:
Not at all - this is super exciting.  What a lovely present to get on a Friday the 13th!  So impressed with the effort 💖 💖

SeanKilleen committed:
Merge branch 'main' into add-spellcheck

SeanKilleen commented:
FYI the easy spelling fixes can now be found in #23216.

SeanKilleen committed:
Merge branch 'main' into pr/23189

SeanKilleen committed:
Merge branch 'main' into add-spellcheck

SeanKilleen commented:
Merged in main to keep it updated. There still might be a few "low-hanging fruit" that I missed; let me know if you want me to separate any remaining items out into another PR and I will.

SeanKilleen committed:
Merge branch 'main' into add-spellcheck

cmwilson21 commented:
@SeanKilleen Thanks again for being flexible and submitting the other PR 🎉 

We are having internal discussions about if/how we can implement this workflow or if it's the right approach for us. That may take some time, but we appreciate the work and the inspiration behind this tool ✨ 

Just wanted to let you know in case it gets closed or moves to an internal repo, we are still talking about it 😄

SeanKilleen commented:
@cmwilson21 no worries! Please have a look at the remaining "low-hanging fruit" and let me know if you'd like me to submit a PR to specifically address any of them.

Also I'd love to understand how this ends up working out internally; if you're willing to share anything, you're welcome to email me at seankilleen@gmail.com. I'm under MS MVP NDA if that helps 😆.

SeanKilleen committed:
Merge branch 'main' into add-spellcheck

SeanKilleen committed:
Fix for new issues: back-ticks

SeanKilleen committed:
Merge branch 'main' into add-spellcheck

SeanKilleen committed:
Merge branch 'main' into add-spellcheck

SeanKilleen committed:
Add word

SeanKilleen committed:
Change some markdown formatting to avoid spellcheck issue

SeanKilleen committed:
Fix spelling error: "registry"

SeanKilleen committed:
Add words

SeanKilleen committed:
fix typo: "repository"

SeanKilleen commented:
FYI @cmwilson21 @martinwoodward I updated this PR which appears to have caught some new spelling errors, so I added changes in this PR for them too. Let me know if you'd like me to address any of them separately.

SeanKilleen committed:
add two known terms

cmwilson21 commented:
@SeanKilleen Thanks for the update! I think separating any new spelling errors into a new PR would be great! This convo is still happening within our team, so while this is still open, any new errors will be reviewed/merged faster in separate PRs.

Thanks for letting us know and checking in on this again! ✨

SeanKilleen committed:
Merge remote-tracking branch 'upstream/main' into add-spellcheck

SeanKilleen committed:
Merge remote-tracking branch 'upstream/main' into add-spellcheck

SeanKilleen committed:
Merge branch 'main' into add-spellcheck

SeanKilleen committed:
Merge remote-tracking branch 'upstream/main' into add-spellcheck

SeanKilleen committed:
Merge branch 'main' into add-spellcheck

SeanKilleen committed:
Merge remote-tracking branch 'upstream/main' into add-spellcheck

SeanKilleen committed:
Merge branch 'main' into add-spellcheck

SeanKilleen committed:
Suggested space to avoid spelling issue being flagged

Logging in this PR rather than a separate PR as it goes to the overall approach

SeanKilleen committed:
Merge remote-tracking branch 'upstream/main' into add-spellcheck

SeanKilleen committed:
Add anchore

SeanKilleen committed:
Fix quote

SeanKilleen committed:
Ignore words

SeanKilleen committed:
Merge remote-tracking branch 'upstream/main' into add-spellcheck

SeanKilleen committed:
Add AUTOTITLE to ignored words

SeanKilleen committed:
fix: "authentication"

SeanKilleen committed:
re-do spacing in liquid syntax

SeanKilleen committed:
Merge branch 'main' into add-spellcheck

SeanKilleen committed:
Merge branch 'main' into add-spellcheck

SeanKilleen committed:
Merge branch 'main' into add-spellcheck

SeanKilleen committed:
Merge branch 'main' into add-spellcheck

SeanKilleen committed:
Merge remote-tracking branch 'upstream/main' into add-spellcheck

SeanKilleen committed:
Add "dotfile" to dictionary

SeanKilleen committed:
fix: add hyphen

SeanKilleen committed:
Merge branch 'main' into add-spellcheck

SeanKilleen committed:
Merge remote-tracking branch 'upstream/main' into add-spellcheck

SeanKilleen committed:
typo: wimdow -> window

SeanKilleen committed:
fix spelling: "administrators"

SeanKilleen committed:
Merge remote-tracking branch 'upstream/main' into add-spellcheck

SeanKilleen committed:
Merge branch 'main' into add-spellcheck

SeanKilleen committed:
Adding "endor" as an ignored phrase

Unsure why -- might actually be a cSpell bug -- but despite adding an explicit regex match (verified with a regex checker), it was still flagged.

SeanKilleen committed:
Merge remote-tracking branch 'upstream/main' into add-spellcheck

SeanKilleen committed:
add "keychains" to dictionary.

SeanKilleen committed:
formatting fix

SeanKilleen committed:
Merge branch 'main' into add-spellcheck

SeanKilleen committed:
Merge branch 'main' into add-spellcheck

SeanKilleen committed:
Merge branch 'main' into add-spellcheck

SeanKilleen committed:
Merge branch 'main' into add-spellcheck

SeanKilleen committed:
Merge branch 'main' into add-spellcheck

SeanKilleen committed:
Add "slugified" to dictionary

It's an intentional choice and it makes sense in context.

SeanKilleen committed:
Merge branch 'main' into add-spellcheck

SeanKilleen committed:
Merge branch 'main' into add-spellcheck

SeanKilleen committed:
ignore "zipball"

SeanKilleen committed:
Merge branch 'main' into add-spellcheck

SeanKilleen committed:
Merge branch 'main' into pr/23189

SeanKilleen committed:
add 3 words

SeanKilleen committed:
fix: sluggified

SeanKilleen committed:
fixes

SeanKilleen committed:
Merge branch 'main' into add-spellcheck

0939368520Thanhphong commented:
0939368520
0915457241

Vào 21:42, Th 2, 13 thg 3, 2023 github-actions[bot] <
***@***.***> đã viết:

> 👋 Hey there spelunker. It looks like you've modified some files that we
> can't accept as contributions. The complete list of files we can't accept
> are:
> .devcontainer/**
> .github/actions-scripts/**
> .github/workflows/**
> .github/CODEOWNERS
> assets/fonts/**
> data/graphql/**
> Dockerfile*
> src/**
> lib/redirects/**
> package*.json
> scripts/**
> content/actions/deployment/security-hardening-your-deployments/**
>
> You'll need to revert all of the files you changed in that list using GitHub
> Desktop
> ***@***.***/desktop/contributing-and-collaborating-using-github-desktop/managing-commits/reverting-a-commit>
> or git checkout origin/main <file name>. Once you get those files
> reverted, we can continue with the review process. [image: :octocat:]
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/github/docs/pull/23189#issuecomment-1466270495>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A4H42WFIHWS5EAD5VFCCOWTW34W4DANCNFSM6AAAAAATZORDNY>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

SeanKilleen committed:
Merge branch 'main' into add-spellcheck

SeanKilleen committed:
Merge branch 'main' into add-spellcheck

SeanKilleen committed:
Merge branch 'main' into add-spellcheck

SeanKilleen committed:
Merge branch 'main' into add-spellcheck

SeanKilleen committed:
fix: "collapsible"

SeanKilleen committed:
Merge remote-tracking branch 'upstream/main' into add-spellcheck

SeanKilleen committed:
Merge branch 'main' into add-spellcheck

SeanKilleen committed:
Merge branch 'main' into add-spellcheck

SeanKilleen committed:
ignore legit word

SeanKilleen committed:
Merge remote-tracking branch 'upstream/main' into add-spellcheck

AS0820075980 commented:
Got it, thanks!

Chaiwat oppo A16

ในวันที่ พฤ. 23 มี.ค. 2023 21:04 น. Alishmshiri ***@***.***>
เขียนว่า:

> ALISHMSHIRY
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/github/docs/pull/23189#issuecomment-1481256711>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A43CNFLAIF3KESPQNADDHO3W5RJ5DANCNFSM6AAAAAATZORDNY>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

SeanKilleen committed:
Merge branch 'main' into add-spellcheck

SeanKilleen committed:
Merge remote-tracking branch 'upstream/main' into add-spellcheck

SeanKilleen committed:
extra stuff

SeanKilleen committed:
spacing

SeanKilleen committed:
backticks

SeanKilleen committed:
typo: "command"

SeanKilleen committed:
add pathnames to dictionary

SeanKilleen committed:
typo: "print"

SeanKilleen committed:
fix: "auto-detecting"

SeanKilleen committed:
adding known words

Even though these ones always sound fake to me :)

SeanKilleen committed:
fix: evaluator

SeanKilleen committed:
add words

SeanKilleen committed:
fix: "evaluator"

SeanKilleen committed:
backticks

SeanKilleen committed:
fix: "superseded"

SeanKilleen committed:
backticks

SeanKilleen committed:
fix: "attempt"

SeanKilleen committed:
backticks

SeanKilleen committed:
fix: "summarized"

SeanKilleen committed:
fix: "evaluator"

SeanKilleen committed:
en-us standardization: "behavior"

SeanKilleen committed:
en-us standardization: "behavior"

SeanKilleen committed:
en-us: analyze

SeanKilleen committed:
en-us standardization: "behavior"

SeanKilleen committed:
fix: "evaluator"

SeanKilleen committed:
fix: "language"

SeanKilleen committed:
fix: configuration

SeanKilleen committed:
fix: specified

SeanKilleen committed:
en-us: analyze

SeanKilleen committed:
fix: "evaluator"

SeanKilleen committed:
add word

SeanKilleen committed:
spacing

SeanKilleen committed:
fix: entity

SeanKilleen committed:
add words

SeanKilleen committed:
fix: configuration

SeanKilleen committed:
fix: specified

SeanKilleen committed:
fix: language

SeanKilleen committed:
words

SeanKilleen committed:
en-us standardization: "behavior"

SeanKilleen committed:
fix frontmatter syntax

SeanKilleen committed:
Merge remote-tracking branch 'upstream/main' into add-spellcheck

SeanKilleen committed:
Merge remote-tracking branch 'upstream/main' into add-spellcheck

SeanKilleen committed:
add word

SeanKilleen committed:
en-US

SeanKilleen committed:
add words

SeanKilleen committed:
backticks

janiceilene commented:
Thanks again for all the amazing work and inspiration @SeanKilleen 😍 We're still working on an internal solution, so I'm going to close this up in the meantime 💖

Modifi-Master commented:
2023-04-20 15:42 GMT+08:00, Janice ***@***.***>:
> Closed #23189.
>
> --
> Reply to this email directly or view it on GitHub:
> https://github.com/github/docs/pull/23189#event-9054130520
> You are receiving this because you are subscribed to this thread.
>
> Message ID: ***@***.***>
***@***.***>
建立時間：	2023年4月19日上午8:16​​ (傳輸時間：0秒)
寄件人：	肖恩·基林 ***@***.***>
收件人：	***@***.***>
IP 192.30.252.201
（嗯！沒差^^

SeanKilleen commented:
@janiceilene I had kept this open in the hopes to learn more about that solution when it happened. I didn't mind submitting fixes in the meantime and this was my primary mechanism to do so. 

Either way, glad to have been able to make a contribution. Good luck with it!
